spec.bs

```diff
@@ -1062,20 +1062,27 @@ To <dfn>encrypt the payload</dfn> given a [=byte sequence=] |plaintextPayload|,
 public key |pkR| and a [=string=] |sharedInfo|, perform the following steps.
 They return a [=byte sequence=] or an error.
 
+1. [=Assert=]: |sharedInfo| is an [=ASCII string=].
 1. Let |info| be the result of [=UTF-8 encoding=] the [=string/concatenation=]
     of « "`aggregation_service`", |sharedInfo| ».
-1. Let (|kem_id|, |kdf_id|, |aead_id|) be (0x0020, 0x0001, 0x0003).
-
-    Note: These indicate the HPKE algorithm identifiers, specifying the KEM
-        function as DHKEM(X25519, HKDF-SHA256), the KDF function as HKDF-SHA256
-        and the AEAD function as ChaCha20Poly1305.
-1. Let |hpkeContext| be the result of setting up an [[RFC9180|HPKE]]
-    [[RFC9180#name-encryption-to-a-public-key|sender's context]] with |pkR|,
-    |info|, |kem_id|, |kdf_id| and |aead_id|.
 1. Let |aad| be \`\` (an empty [=byte sequence=]).
-1. Let <var ignore>encryptedPayload</var> be the result of
-    [[RFC9180#name-encryption-and-decryption|encrypting]] |plaintextPayload|
-    with |hpkeContext| and |aad|.
+1. Let |ciphersuite| be (0x0020, 0x0001, 0x0003).
```

Comment:
very optional: the names kem_id, kdf_id and aead_id might align slightly better with the HPKE spec, but either way let's definitely keep the links to each section in the Note.

Reply:
Fair point, let's do that.